core/sail/fts/lucene-api/src/main/java/org/eclipse/rdf4j/sail/lucene/LuceneSail.java

```diff
@@ -252,6 +257,33 @@
 	public static final String INCOMPLETE_QUERY_FAIL_KEY = "incompletequeryfail";
 
 	/**
+	 * Set this key to "false" to disable late evaluation of search queries.
```

Comment:
Not sure I understand. Is this constant the name of the config property? If so what are its possible values?